frontend/src/layout/navigation-3000/navigationLogic.tsx

```diff
@@ -329,6 +335,30 @@ export const navigation3000Logic = kea<navigation3000LogicType>([
                             icon: <IconDashboard />,
                             logic: isUsingSidebar ? dashboardsSidebarLogic : undefined,
                             to: isUsingSidebar ? undefined : urls.dashboards(),
+                            sideAction: {
+                                identifier: 'pinned-dashboards-dropdown',
+                                dropdown: {
+                                    overlay:
+                                        dashboardsLoading || true ? (
+                                            <div className="w-50 px-2 py-1">
+                                                <Spinner textColored /> Loading…
+                                            </div>
+                                        ) : (
+                                            <div className="w-50">
```

Comment:
I intentionally used a fixed width since I thought it looked better with smaller dashboard titles & would prevent a visual jump in case dashboards are still loading (unlikely I guess).

Reply:
Removed with the refactor to `LemonMenuOverlay`

Reply:
Or, it's totally fine actually. Let's keep as is